tests/test_model.py

```diff
@@ -352,13 +353,15 @@ def test_loss_fn():
         pytest.skip('Fused cross entropy was not installed')
 
     # run numerical test in pure fp32
-    torch.backends.cuda.matmul.allow_tf32 = False
-    torch.backends.cudnn.allow_tf32 = False
+    torch.backends.cuda.matmul.allow_tf32 = False  # type: ignore (third-party)
+    torch.backends.cudnn.allow_tf32 = False  # type: ignore (third-party)
 
     conf_path = 'scripts/train/yamls/pretrain/testing.yaml'
     with open(conf_path) as f:
         test_cfg = om.load(f)
 
+    assert isinstance(test_cfg, DictConfig)
+
```

Comment:
```suggestion
    with open(conf_path) as f:
        test_cfg: DictConfig = om.load(f)

```

Would that accomplish the same thing?

Reply:
No, that would fail the checker as `om.load` can also return a `ListConfig`. 

The assert here is not just to make the typechecker happy -- it also plays a valid input checking role in making sure that user configs are actually a dict and not a list at the top level.